spec/scryfall/cards_spec.rb

```diff
@@ -6,7 +6,11 @@
   context 'search' do
     it 'returns a JSON response containing the search results' do
       VCR.use_cassette('successful search') do
+<<<<<<< HEAD
+        expect(Scryfall::Cards.search('c:c cmc=15')['data'].length).to eq(2)
+=======
         expect(Scryfall::Cards.search(query: 'c:c cmc=15')['data'].length).to eq(2)
+>>>>>>> efaa00bfd662bf62bd468c115598a4e4972a2f81
```

Comment:
This sort of garbage gets created when merge conflicts don't get fixed correctly.  You can see there are two versions of the test line above, one from your local branch and one from main.  Before you make your own branch for a feature, you should always do `git checkout main` and `git pull`.  That way you'll have the most updated code to start your new branch.  If changes are made to main after you have started your branch, then you can do `git checkout main`, `git pull`, `git checkout <my-branch>`, `git merge main`, and `git push`.  This should be done while your branch is clean, so if you have pending changes you haven't committed, but still need to update your branch with main, then you can do `git stash` before all of those commands above and then `git stash apply` afterward.  That will remove your changes and then restore them.